src/open_inwoner/conf/base.py

```diff
@@ -648,6 +648,51 @@
             {"name": "email", "description": _("Email of the invited user")},
         ],
     },
+    "contact_approval": {
+        "name": _("Contact Approval Email"),
+        "description": _(
+            "This email is used to notify people for pending approvals of new contacts"
+        ),
+        "subject_default": "Approval for {{ site_name }}",
+        "body_default": """
+            <p>Beste</p>
+
+            <p>User {{ sender_name }} has requested to be your contact.
+            Follow the link below to approve or reject the connection </p>
+
+            <p><a href="{{ contacts_link }}">My contacts</a> </p>
+
+            <p>The notification will remain on your contact list page until your approval/rejection </p>
+
+            <p>Sincerely,
+            {{ site_name }} </p>
```

Comment:
Beste,

Gebruiker {{ sender_name }} wilt u toevoegen als contactpersoon op {{ site_name }}. Volg onderstaande link waarop u uw goedkeuring kan geven of kan aangeven {{ sender_name }} niet als contactpersoon te willen.

Mijn Contacten

U kunt ook op een later moment toestemming geven, het verzoek van {{ sender_name }} blijft open staat totdat u een keuze heeft gemaakt.

Met vriendelijke groet,
{{ site_name }}